src/Miunie.Storage/JsonDataStorage.cs

```diff
@@ -76,39 +70,24 @@ public void WipeData()
             Directory.Delete(_resourcesFolder);
         }
 
-        public bool KeyExists(string collection, string key)
-        {
-            var file = GetFileNameByKey(key);
-            return LocalFileExists(String.Concat(collection, "/", file));
-        }
+        private string KeyToFullFileName(string key) => String.Format(FileTemplate, key);

```

Comment:
In the rest of the project, we use a different style for expression-bodied members, starting with `=>` in a new line, just like this:
```cs
private string KeyToFullFileName(string key) 
    => String.Format(FileTemplate, key);
```
So, just for consistency, let's change it.